docs/containers-storage.conf.5.md

```diff
@@ -89,9 +89,9 @@ Tells driver to wipe device (directlvm_device) even if device already has a file
 
 Specifies the filesystem type to use for the base device. (default: xfs)
 
-**fuse_program**=""
+**mount_program**=""
 
-Specifies the path to a custom FUSE program to use instead for mounting the file system.
+Specifies the path to a custom program to use instead for mounting the file system.
```

Comment:
This shouldn't be in the `storage.options.thinpool` section.

Reply:
thanks, I've moved it